TravelerApp/TravelerAppCore/Models/Hotels/Hotel.cs

```diff
@@ -0,0 +1,200 @@
+﻿using System;
+using System.Collections.Generic;
+using TravelerAppCore.Controller;
+
+namespace TravelerAppCore.Models.Hotels
+{
+    public class Hotel
+    {
+        public List<Review> Reviews { get; set; }
+        public HotelInfo HotelInfo { get; set; }
+
+        private string log;
+        private Ratings averages;
+        public Ratings AverageRates
+        {
+            get
+            {
+                float averageOverall = 0;
+                int overall = 0;
+                float averageService = 0;
+                int service = 0;
+                float averageCleanliness = 0;
+                int cleanliness = 0;
+                float averageValue = 0;
+                int value = 0;
+                float averageSleepQuality = 0;
+                int sleepQuality = 0;
+                float averageRooms = 0;
+                int rooms = 0;
+                float averageLocation = 0;
+                int location = 0;
+
+                foreach (Review item in Reviews)
+                {
+
+                    if (item.Ratings.Overall != 0)
```

Comment:
czy jak klient oceni hotel na 0 to ma to się nie wliczać do średniej?

Reply:
0 oznacza brak oceny. Zmieniłem skalę na 1-5